public/Aryan Suri/Sidebar/sidebar.js

```diff
@@ -594,871 +653,773 @@ function createSidebar() {
+      document.getElementById("sidebar3").style.width = "480px";
+      document.getElementById("sidebar2").style.width = "0px";
+
+      //document.getElementById("main").style.marginLeft = "250px";
+    }
+  });
+
+  window.addEventListener('click', function (event) {
+    if (event.target == document.getElementById("developer_home")) {
+      document.getElementById("sidebar3").style.width = "0";
+      document.getElementById("custom_sidebar").style.width = "480px";
+
+
+      //document.getElementById("main").style.marginLeft = "250px";
+    }
+  });
+
+  window.addEventListener('click', function (event) {
+    if (event.target == document.getElementById("developer_acess")) {
+      document.getElementById("sidebar1").style.width = "480px";
+      document.getElementById("sidebar3").style.width = "0px";
+
+      //document.getElementById("main").style.marginLeft = "250px";
+    }
+  });
+
+  window.addEventListener('click', function (event) {
+    if (event.target == document.getElementById("developer_refer")) {
+      document.getElementById("sidebar2").style.width = "480px";
+      document.getElementById("sidebar3").style.width = "0px";
+
+      //document.getElementById("main").style.marginLeft = "250px";
+    }
+  });
+
+  window.addEventListener('click', function (event) {
+    if (event.target == document.getElementById("open_usage")) {
+      document.getElementById("sidebar4").style.width = "480px";
+      document.getElementById("custom_sidebar").style.width = "0px";
+
+      //document.getElementById("main").style.marginLeft = "250px";
+    }
+  });
+
+  window.addEventListener('click', function (event) {
+    if (event.target == document.getElementById("usage_home")) {
+      document.getElementById("custom_sidebar").style.width = "480px";
+      document.getElementById("sidebar4").style.width = "0px";
+
+      //document.getElementById("main").style.marginLeft = "250px";
+    }
+  });
+
+  window.addEventListener('click', function (event) {
+    if (event.target == document.getElementById("usage_acess")) {
+      document.getElementById("sidebar1").style.width = "480px";
+      document.getElementById("sidebar4").style.width = "0px";
+
+      //document.getElementById("main").style.marginLeft = "250px";
+    }
+  });
+
+  window.addEventListener('click', function (event) {
+    if (event.target == document.getElementById("usage_refer")) {
+      document.getElementById("sidebar2").style.width = "480px";
+      document.getElementById("sidebar4").style.width = "0px";
+
+      //document.getElementById("main").style.marginLeft = "250px";
+    }
+  });
+
+  window.addEventListener('click', function (event) {
+    if (event.target == document.getElementById("usage_developer")) {
+      document.getElementById("sidebar3").style.width = "480px";
+      document.getElementById("sidebar4").style.width = "0px";
+
+      //document.getElementById("main").style.marginLeft = "250px";
+    }
+  });
```

Comment:
We will handle this later, but these actually seem like something we could consolidate into a relatively simple function. Remind me after this is launched for us to go back and clean this up, since it will make any future changes easier (such as adding new tabs).

Reply:
Also, is there a reason why custom_sidebar has a different naming scheme than sidebar1, sidebar2 etc?